lib/bolt/plugin.rb

```diff
@@ -235,16 +237,16 @@ def get_hook(plugin_name, hook)
 
     # Calling by_name or get_hook will load any module based plugin automatically
     def by_name(plugin_name)
-      return @plugins[plugin_name] if @plugins.include?(plugin_name)
-      begin
-        if RUBY_PLUGINS.include?(plugin_name)
+      if known_plugin?(plugin_name)
+        if @plugins.include?(plugin_name)
```

Comment:
This is a question for my own understanding more so than a review comment: Is the reason this `@plugins.include?` happens before the `!@load_plugins` check to allow us to load plugins even when it's technically disabled?

Reply:
Does this have implications for users where they could end up loading plugins even with plugin loading disabled? I guess, is plugin loading something that they even have control over in the first place?

Reply:
No, this only affects bolt-server's ability to explicitly load plugins. From a bolt user's point of view, all plugins are loaded magically and there's no way in bolt to disable plugin loading at all.